helm-chart/values.yaml

```diff
@@ -155,3 +136,20 @@ readinessProbe:
     port: jsonrpcrelay
   initialDelaySeconds: 20
   timeoutSeconds: 5
+
+test:
+  enabled: true
+  image:
+    pullPolicy: IfNotPresent
+    repository: postman/newman
+    tag: 5.3.1-alpine
+  postman: ""  # Custom postman.json in base64 encoding
+  schedule: '@daily'
+
+terminationGracePeriodSeconds: 60
```

Comment:
This is being set to 1 elsewhere [https://github.com/hashgraph/hedera-json-rpc-relay/pull/1149#discussion_r1183912996] and the value here is not used by the chart.  Lets change this to 1 and use the value defined here in the tests/pod.yaml definition

Reply:
Gotcha, thanks for explaining!  👍